config/footer.app

```diff
@@ -1608,3 +1600,21 @@ export REPORT_VARIANTS_FULL
 
 # TEST
 #THREADS=16
+
+#RULES export has to be after PIPELINE is defined and no longer changed, otherwise some rules might not be loaded
+steps_without_rule_file="sorting compress"
+RULES=$(ls $STARK_FOLDER_RULES/*.mk | tr '\n' ' ')
```

Comment:
Pourquoi pas *.rules.mk ?

Reply:
A cause de config.mk ; alternativement on peut le renommer (à moins qu'il n'y ait pas besoin de charger celui là ?)

Reply:
Ce config.mk est géré et chargé à part dans NGSWorkflow.mk
Tu peux mettre *.rules.mk ?

Reply:
Done in #e6cb109